golden-tests/GoldenTests.hs

```diff
@@ -25,6 +25,11 @@ import DhallLocation ( DhallLocation ( DhallLocation ) )
 import DhallToCabal ( dhallToCabal )
 
 
+
+-- Note: all of the dhall-to-cabal tests use the CWD as their root
+-- directory for imports, not the directory that they live in.
```

Comment:
I actually think this is confusing, and if we're going to fix up cwd stuff, let's do it properly and have all those files have correct references.